_examples/builtin_array/builtin_array.go

```diff
@@ -0,0 +1,18 @@
+package builtin_array
```

Comment:
`s/builtin_array/slices/`

we are testing slices, not arrays. (in the Go sense).
and rename the parent directory too :)